.drone.jsonnet

```diff
@@ -169,6 +176,8 @@ local Pipeline(branch, platform, event) = {
                "sed -i -e 's/\"galera-enterprise-4\"//' cmake/cpack_rpm.cmake",
                "sed -i '/columnstore/Id' debian/autobake-deb.sh",
                "sed -i 's/.*flex.*/echo/' debian/autobake-deb.sh",
+               "sed -i 's/BETA/GAMMA/' storage/columnstore/CMakeLists.txt",
```

Comment:
@pleblanc1976 The suggested way look inappropriate b/c neither me nor you don't know what happens when mysql_add_plugin() got called. We better change configuration setting plugin_maturity_level to allow beta quality plugins to be installed prior the plugin installation.